src/store/messagesStore.js

```diff
@@ -504,7 +504,7 @@ const mutations = {
 			return
 		}
 
-		const messagesToRemove = messageIds.sort().reverse().slice(199)
+		const messagesToRemove = messageIds.map(parseInt).sort((a, b) => b - a).slice(199)
```

Comment:
```js
messageIds.map(parseInt)
```
is the same as
```js
messageIds.map((id, index) => parseInt(number=id, radix=index))
```

So `['9', '10', '11', '12'].map(parseInt)` doesn't return `[9, 10, 11, 12]`. It returns `[9, NaN, 3, 5]` using index as radix.

Reply:
`parseInt` is not needed, `b - a` already casts to numbers

Reply:
Yeah, but we need to compare it later on line 514 (though it would be easier to cast a firstKnown as a string there)